### PR TITLE
Fix metamask white screen bug

### PR DESCRIPTION
**Pull request by derivationstaff001**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+theServer=df-prod
+theFolder=/var/www/html/app-test/
+buildEnv=dev
+
+case "$1" in
+    "prod")
+        theFolder=/var/www/html/mobile/
```
Comment (LiyueWang) [inline deploy.sh:7]: @derivationstaff001 在两个文件地址也是，都确认过？回头你可以把前端在服务器的部署情况和配置情况整理成一个文档放

Comment (LiyueWang) [inline deploy.sh:7]: @derivationstaff001 check the folders here too,  and doc them in [derify-docs-internal](https://github.com/derivationlab/derify-docs-internal)

Comment (LiyueWang) [reply deploy.sh:7]: tip: some sensitive configs better to be stored in private config file instead of the open souce code

Comment (derivationlab) [inline deploy.sh:7]: @derivationstaff001 check the folders here too,  and doc them in [derify-docs-internal](https://github.com/derivationlab/derify-docs-internal)

Comment (derivationlab) [reply deploy.sh:7]: tip: for some sensitive configs better to be stored in private config file instead of the open source code

**Review by derivationlab — APPROVED** [review]

File: deploy.sh (hunk 1 above)
Comment (derivationstaff001) [reply deploy.sh:7]: these should be in CI/CD system. but we do not use them yet...
if we use vercle.com, or cloudflare have similar function then we event do not need to write any shell script to do this.

**Pull request merged**